sources/packages/backend/libs/services/src/restriction/model/restriction.model.ts

```diff
@@ -33,4 +33,19 @@ export enum RestrictionCode {
    * reached. It is calculated during E-cert.
    */
   BCLM = "BCLM",
+  /**
```

Comment:
I believe the intention of the restriction codes existing in the enum is to be used in the backend logic.

I would say not to extend them if they are used for e2e test purpose only. Rather, create an enum in test lib with all the extended properties.